tools/build-release-tarball

```diff
@@ -83,7 +86,7 @@ find prod-static/serve/third/gemoji/images/emoji/ -regex '.*\.[0-9a-f]+\.png' -d
 
 cd "$TMPDIR"
 
-tar --append -f "$TARBALL" "$prefix/prod-static" "$prefix/build_id" "$prefix/version" "$prefix/locale"
+tar --append -f "$TARBALL" "$prefix/prod-static" "$prefix/build_id" "$prefix/version" "$prefix/locale" "$prefix/var"
```

Comment:
We don't want the `var` stuff in the tarball, so you should drop this line.  Also, can you rebase this into the commit that introduced `var/log` for better clarity?